Sources/DistributedActors/Cluster/ClusterShell.swift

```diff
@@ -277,33 +284,17 @@ internal class ClusterShell {
     }
 
     /// Actually starts the shell which kicks off binding to a port, and all further cluster work
-    internal func lazyStart(system: ClusterSystem, clusterEvents: EventStream<Cluster.Event>) throws -> LazyStart<Message> {
```

Comment:
This doesn't look like it gets used, so removing it.

Reply:
nice :)